script.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-unused-vars */
+// Book Class: Represents a Book
+// eslint-disable-next-line max-classes-per-file
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// Store Classs: Hundles Storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(author) {
+    const books = Store.getBooks();
+    books.forEach((book, index) => {
+      if (book.author === author) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
```

Comment:
- [ ] [OPTIONAL] I will suggest giving every book created a unique id so that it becomes easy to remove using that `id` instead of `author`. This is because more than one book can have the same authors, then bugs can be introduced if you try removing them.